PSR2R/Sniffs/Classes/InterfaceNameSniff.php

```diff
@@ -11,18 +12,12 @@ class InterfaceNameSniff extends AbstractSniff {
 
 	/**
 	 * @inheritDoc
+	 * @return void
```

Comment:
Looks like it is PSR2R.Commenting.DocBlockReturnTag.Missing. I'll fix the sniff.